src/components/Select.jsx

```diff
@@ -13,14 +13,25 @@ const Select = ({
     <div className="mt-3">
       <p className="mb-1">{title}</p>
       <Dropdown className="w-full m-0">
-        <Dropdown.Toggle
-          id="dropdown-toggle"
-          className={`!bg-white ${
-            user[field] ? "text-black" : "!text-hackathon-placeholder"
-          } w-full !text-left !border-x-0 !border-t-0 !border-b-2 !rounded-none !border-black`}
-        >
-          {user[field] || placeholder}
-        </Dropdown.Toggle>
+        {editable ? (
```

Comment:
see if there is a way to keep the dropdown toggle but just remove the icon and its functionality conditionally

Reply:
but when you click the pencil, the dropdown toggle comes back and conditionally operates only when in edit mode

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/49844738/e2a912e3-a1ca-4d10-826b-74515a99f7fe)